boot/zephyr/main.c

```diff
@@ -277,8 +277,6 @@ static void copy_img_to_SRAM(int slot, unsigned int hdr_offset)
  */
 static void do_boot(struct boot_rsp *rsp)
 {
-    void *start;
```

Comment:
variables should be declared at start of function so this needs to stay, you can #ifdef around it here